modules/nw-egress-ips-namespaces.adoc

```diff
@@ -0,0 +1,68 @@
+// Module filename: nw-egress-ips-namespaces.adoc
+//
+// Module included in the following assemblies:
+// * networking/managing-networking.adoc
+
+[id='nw-egress-ips-namespaces-{context}']
+= Enabling Automatic Egress IPs
+
+As a cluster administrator, you can assign
+egress IP addresses to namespaces by setting the `egressIPs` field to the
+`NetNamespace` resource.
+
+With fully automatic egress IPs, you can set the `egressCIDRs` field
+of each node's `HostSubnet` resource to indicate the range of egress
+IP addresses that can be hosted.
+Namespaces that have requested egress IP addresses are matched
+with nodes that are able to host those egress IP addresses, then the egress IP
+addresses are assigned to those nodes.
+
+High availability is automatic. If a node hosting egress IP addresses
+goes down and there are nodes that are able to host those egress IP addresses,
+based on the `egressCIDR` values of the `HostSubnet` resources, then the egress
+IP addresses move to a new node. When the original egress IP address node
+comes back online, the egress IP addresses automatically move to balance
+egress IP addresses across nodes.
+
+[IMPORTANT]
+====
+You cannot use manually assigned and automatically assigned egress IP addresses
+on the same nodes or with the same IP address ranges.
+====
+
+.Procedure
+
+. Update the `NetNamespace` with the egress IP address:
```

Comment:
NetNamespace resource

Reply:
Applied in https://github.com/openshift/openshift-docs/pull/14555.